bodhi/server/util.py

```diff
@@ -315,24 +316,54 @@ def sanity_check_repodata(myurl, repo_type):
 
         if repo_type in ('yum', 'source'):
             tests.append((['list', 'available'], 'testrepo'))
+            tests.append((['updateinfo'], '.*'))
         elif repo_type == 'module':
             tests.append((['module', 'list'], '.*'))
 
+        cfg_options = {
+            '--setopt=zchunk': ['0', '1'],
+        }
+
+        cfg_combinations = build_cfg_combinations(cfg_options)
+
         for test in tests:
             dnfargs, expout = test
 
-            # Make sure every DNF test runs in a new temp dir
-            testdir = tempfile.mkdtemp(dir=tmpdir)
-            output = sanity_check_repodata_dnf(testdir, myurl, *dnfargs)
-            if (expout == ".*" and len(output.strip()) != 0) or (expout in output):
-                continue
-            else:
-                raise RepodataException(
-                    f"DNF did not return expected output when running test!"
-                    + f" Test: {dnfargs}, expected: {expout}, output: {output}")
+            for config in cfg_combinations:
+                config = list(config)
+
+                # Make sure every DNF test runs in a new temp dir
+                testdir = tempfile.mkdtemp(dir=tmpdir)
+                output = sanity_check_repodata_dnf(testdir, myurl, config, *dnfargs)
+                if (expout == ".*" and len(output.strip()) != 0) or (expout in output):
+                    continue
+                else:
+                    raise RepodataException(
+                        f"DNF did not return expected output when running test!"
+                        + f" Test: {dnfargs}, config: {config}, expected: {expout},"
+                        + f" output: {output}")
+
+
+def build_cfg_combinations(options):
+    """
+    Builds the Cartesian product of cfg_options
```

Comment:
`cfg_options` should probably be `options` here, since it's the function's arg name.